geode-core/src/main/java/com/gemstone/gemfire/management/internal/cli/AbstractCliAroundInterceptor.java

```diff
@@ -88,7 +88,7 @@ protected Response readYesNo(String message, Response defaultResponse) {
       try {
         String userInput = interact(message);
 
-        if (userInput == null || userInput == "") {
+        if (userInput.isEmpty()) {
```

Comment:
Here you need to keep the null check or it could throw an NPE, so:
  if (userInput == null || userInput.isEmpty()) {